docker-compose.yaml

```diff
@@ -1,7 +1,6 @@
 services:
     frontend:
-        build:
-            context: .
+        image: ghcr.io/accenture/federated-catalogue-viewer:latest
```

Comment:
Since this is meant for local use I would rather keep the build context. Or both, and add a small hint in the docs to use the `--build` flag to build it locally.

Reply:
[26b2853](https://github.com/Accenture/Federated-Catalogue-Viewer/pull/2/commits/26b2853fadd686ffe11c5d7dbe1cc082ebfa1ecf)